package.json

```diff
@@ -38,6 +38,7 @@
     "@trpc/react": "^4.0.0",
     "@trpc/server": "^4.0.0",
     "countries-list": "^2.6.1",
+    "fishery": "^1.2.0",
```

Comment:
I believe this should be under devDependencies since we won't need it in production: (so just running `npm i fishery -D` should do the trick)